ihontaryk/homework_8/restaurant.py

```diff
@@ -0,0 +1,84 @@
+from collections import OrderedDict
+from collections import namedtuple
+
+
+class Restaurant:
+    menu = {'first_course': 'First Course',
+            'second_course': 'Second Course',
+            'dessert': 'Dessert',
+            'beverage': 'Beverage'}
+
+    def __init__(self):
+        self.order = OrderedDict()
+
+    def _add_item(self, key, count, price):
+        if key not in self.menu:
+            raise ValueError(f'{key} is not available in the menu')
+
+        name = self.menu[key]
+
+        if name not in self.order:
+            self.order[name] = 0
+        self.order[name] += count * price
+
+    def get_client_order(self, *order: tuple):
```

Comment:
def get_client_order(self, *order: Order):


it is not tuple because you use its attrs